lib/core/configuration.dart

```diff
@@ -68,6 +69,8 @@ class Configuration {
   static const keyShouldShowLockScreen = "should_show_lock_screen";
   static const keyHasSelectedAnyBackupFolder =
       "has_selected_any_folder_for_backup";
+  static const keyHasEnabledTwoFactor = "has_enabled_two_factor";
```

Comment:
Should we move this config key and logic insider UserService?  Configuration is already doing too many things which it should not be doing :)